client/src/connection.ts

```diff
@@ -224,6 +224,7 @@ export class ConnectionManager implements IConnectionManager {
 
     if ((this.connectionConfiguration.host == undefined) || (this.connectionConfiguration.host == '')) {
       args.push('--ip=127.0.0.1');
+      args.push('--local-workspace=' + vscode.workspace.textDocuments[0]);
```

Comment:
Ya, that's why I chose textDocuments. It looked like it'd be populated with all the files of a workspace. So just pick one and work from there to metadata.json